src/features/rules/tests/createSelector.test.js

```diff
@@ -0,0 +1,21 @@
+import { createSelector } from "../createSelector";
```

Comment:
We already have `__tests__` and `__mocks__` directories. Why can't we put these tests there?

Reply:
sure, moved to a common folder